pretest.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+require('./test/init.js');
+
+var async = require('async');
+var fs = require('fs');
+var path = require('path');
+var _ = require('lodash');
+
+var db = global.getDataSource();
+var schemaName = global.config.schema;
+var sqlFile = path.resolve(__dirname, 'test', 'tables.dashdb.sql');
+
+if (!process.env.CI)
+  return console.log('Skipping pre-test ...');
+console.log('> Seeding the database ...');
+
+var contents = fs.readFileSync(sqlFile, {encoding: 'utf-8'});
+var lines = contents.trim().split(';');
+var createView = `CREATE OR REPLACE VIEW "${schemaName}"."INVENTORY_VIEW" AS
+  SELECT P.name AS product, L.name AS location, I.available FROM
+    "${schemaName}"."INVENTORY" I,
+    "${schemaName}"."PRODUCT" P,
+    "${schemaName}"."LOCATION" L
+    WHERE
+    p.id = I.product_id AND l.id = I.location_id;`;
+
+// create a schema with the build number
+db.connector.execute('CREATE SCHEMA "' + schemaName + '";',
```

Comment:
same as posttest, I am still worried will the main callback function of `async.each` wait until you execute all async code.

```js
async.each(collection, processItem, function(err, result) {
    asyncFunction();
})
```